src/2048-expert/2048-expert.css

```diff
@@ -88,12 +88,30 @@ h1 {
  .game-board div {
    width: 80px;
    height: 80px;
+   line-height: 80px;
    margin: 6px;
    border-radius: 3px;
    font-weight: bold;
    text-align: center;
-   font-size: 60px;
-   line-height: 1.5;
    background-color: #eee4da;
    color: #afa192;
+ }
+
+ /* add colour to cells */
+ [data-colour="0"] {
```

Comment:
`data-colour` --> `data-digits`

I think this is more descriptive of how the cell type is categorised. I think you could also merge `data-colour="0"` and `data-colour="1"` together since they both are used to style 1 digit numbers.